packages/terra-application-name/src/ApplicationMenuName.scss

```diff
@@ -0,0 +1,24 @@
+/* stylelint-disable selector-class-pattern */
+:local {
+  .application-menu-name {
+    align-items: var(--terra-application-menu-name-align-items, center);
```

Comment:
Looking at the screnshots, I am wondering if we need to add a width or min-width to get the centered behavior.